os/board/rtl8730e/src/component/soc/amebad2/fwlib/rom_common/ameba_assert_rom.c

```diff
@@ -11,7 +11,7 @@
 __weak HAL_ROM_TEXT_SECTION
 void io_assert_failed(uint8_t *file, uint32_t line)
 {
-	DBG_8195A("io driver parameters error! file_name: %s, line: %d", file, line);
+	dbg("io driver parameters error! file_name: %s, line: %d", file, line);
 
-	for (;;);
+	ASSERT(0);
 }
```

Comment:
ASSERT also prints __FILE__ and __LINE__.
```
#define ASSERT(f) \
	{ if (!(f)) up_assert((const uint8_t *)__FILE__, (int)__LINE__); }
```
How about remapping the function to ASSERT to reduce code size?
```
#define io_assert_failed(f, l) ASSERT(0)
```